### PR TITLE
Node patterns

### PR DESCRIPTION
**Pull request by werebus**
A rewrite attempt that makes use of [node patterns](https://docs.rubocop.org/en/latest/node_pattern/) to find the offending nodes. Happens to also address the same failure as #6.

### CODE REVIEW

File: umts-custom-cops.gemspec [hunk 1]
```diff
@@ -22,16 +21,14 @@ Gem::Specification.new do |spec|
   end
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(spec)/}) }
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
```
Comment (Anbranin) [inline umts-custom-cops.gemspec:26]: Why these changes? Just curious.

**Review by Anbranin — APPROVED** [review]

File: umts-custom-cops.gemspec (hunk 1 above)
Comment (werebus) [reply umts-custom-cops.gemspec:26]: Rubocop had some other opinions about the gemspec, and while I was cleaning up, I removed these two lines because they're not used.
